README.rst

```diff
@@ -58,6 +58,7 @@ Requirements
 ############
 
 - kytos/of_core
+- Kytos/flow_manager
```

Comment:
You are starting Kytos/flow_manager with upper case "K" here and the next lines that use the "kytos/napp" notation.

Reply:
Resolved.